packages/world/mud.config.ts

```diff
@@ -378,6 +378,16 @@ export default defineWorld({
       key: [],
     },
     // ------------------------------------------------------------
+    // Farming
+    // ------------------------------------------------------------
+    SeedGrowth: {
+      schema: {
+        entityId: "EntityId",
+        fullyGrownAt: "uint128",
```

Comment:
not sure what our patterns are here so feel free to ignore, but I found in some projects I was fiddling with that it was a bit more flexible to use something like `plantedAt` and then adjust the time-to-grow instead of combining the time-to-grow with the planted-at timestamp

Reply:
I don't think we have spent too much effort yet on trying to follow a consistent set patterns, I was hoping that after we finish the main features we could do a pass where we analyze the schemas and try to refactor as much as we can for efficiency and consistency.

Having said that, I would love to better understand the benefits in flexibility you mention, as originally I had used plantedAt and timeToGrow for each seed, but ended up with just fullyGrownAt to make it a bit cheaper (and if we need to get planted at we can just do fullyGrownAt - timeToGrow).

Reply:
fullyGrownAt - timeToGrow wouldn't be accurate if you adjust the timeToGrow constant after setting fullyGrownAt in storage

Reply:
Yeah agreed, maybe the optimization is not worth it, however if we in the future wanted to adjust timeToGrow we would need to update for all seeds, the idea was to be able to have something like fertilizer so that you decrease the time to grow. With fullyGrownAt you can still do this if you assume the initial timeToGrow is constant.

Reply:
ahh, modifiers makes sense

in the case of fertilizer, would you be decreasing the `fullyGrownAt`? and would you still have to track if fertilizer was used or not to avoid multiple uses? or maybe each use is a multiplier (e.g. `* 0.5`) rather than subtracting a constant, so it has diminishing returns and rarely reach zero with just fertilizer

Reply:
yeah something like that. Another option would be to store both plantedAt + timeToGrow, and just adjust timeToGrow when using fertilizer. I guess it's fine if we adjust the initial timeToGrow constant without updating all the existing trees, and just apply them for future ones.